docs/src/index.md

```diff
@@ -7,6 +7,22 @@ representations through different "view" types, and to provide some
 useful low-level functions (including "traits") that simplify image
 display, input/output, and the writing of algorithms.
 
+Some of the key features and functionalities provided by ImageCore.jl include:
+
+1.Image I/O: Loading and saving images in various formats like PNG, JPEG, BMP, TIFF, etc.
+2.Image data representation: It provides the Image type to store image data along with functions for basic image manipulation.
+3.Resizing and scaling: Functions for resizing and scaling images.
+4.Image rotation and flipping: Functions to rotate and flip images.
```

Comment:
Resizing, scaling, image rotation and flipping are provided by ImageTransformations.jl and exported there